main.tf

```
@@ -43,7 +43,10 @@ resource "github_repository" "default" {
   }
 
   lifecycle {
-    ignore_changes = [pages]
+    ignore_changes = [
+      pages,
+      template
+    ]
```

```suggestion
    ignore_changes = [pages, template]
```

Bit friendlier on the eye.